Client/app/src/main/java/ru/hse/client/groups/ListChapterData.kt

```diff
@@ -0,0 +1,7 @@
+package ru.hse.client.groups
+
+class ListChapterData (
+    var number: Int,
+    var name: String,
+    var tests: Int
+)
```

Comment:
it could be moved to ListChapterAdapter, it will be more correct, and decrease number of files

Reply:
I maintain uniformity, I do it the same way as Roman did, so that it is convenient to watch